src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -90,7 +90,6 @@
         "symfony/property-info": "<4.4",
         "symfony/property-access": "<5.3",
         "symfony/serializer": "<5.2",
-        "symfony/service-contracts": ">=3.0",
```

Comment:
Given that most projects are probably relying on the transitive for that one and extra.symfony.require won't help in Flex here (it might transitively due to the DI component), is it actually safe to remove the conflict rule ?

Reply:
I think it is. On a Symfony 5.4 + Flex project, the DI component will pin the contracts to v2 anyway.